.ci/Jenkinsfile

```diff
@@ -483,4 +536,11 @@ def isPackageEnabled() {
 */
 def isExtendedWindowsEnabled() {
   return env.EXT_WINDOWS_CHANGES == "true" || params.extended_windows_ci || env.GITHUB_COMMENT?.contains('extended windows') || matchesPrLabel(label: 'ci:extended-windows')
-}
\ No newline at end of file
+}
+
+/**
+* Wrapper to know if the build should enable the M1 extended support
+*/
+def isExtendedM1Enabled() {
+  return env.EXT_M1_CHANGES == "true" || params.extended_m1_ci || env.GITHUB_COMMENT?.contains('extended m1') || matchesPrLabel(label: 'ci:extended-m1')
```

Comment:
If I'm follow this correctly, m1 only runs when specifically request or if we change the Jenkinsfile (from `setEnvVar('EXT_M1_CHANGES', isGitRegionMatch(patterns: [ '.ci/Jenkinsfile' ], shouldMatchAll: false).toString())`)?

Reply:
That's correct, the `EXT_M1_CHANGES` env variable could be changed to something else if needed, so far, I added the `.ci/Jenkinsfile` to the list of files that should enable this extended validation. 

At the moment there are 4 different reasons the stage will run on a PR basis:
- If a GitHub comment with `run extended m1`
- If a GitHub label `ci:extended-m1`
- If a Build using the jenkins UI with the parameter `extended_m1_ci` set true.
- If a change in the file `.ci/Jenkinsfile`